src/js/Controllers/DisplayCapabilities.js

```diff
@@ -689,8 +712,9 @@ let capabilities = {
 			"imageCapabilities": ["DYNAMIC", "STATIC"],
 			"menuLayoutsAvailable": ["LIST", "TILES"]
 		},
-		"softButtonCapabilities": []
-	},
+		"softButtonCapabilities": [],
+		"buttonCapabilities": commonButtonCapabilities
+ 		},
```

Comment:
```suggestion
	},
```
Seems some whitespace got added here

Reply:
@jacobkeeler please find fix [here](https://github.com/LuxoftSDL/generic_hmi/commit/cb0bf3ce6f4b47c6c78056efcd1b5fb02fe1be69)